docker-entrypoint.sh

```diff
@@ -41,3 +38,5 @@ elif [[ "$DEV_SERVER" = "1" ]]; then
 else
     uwsgi --ini .prod/uwsgi.ini
 fi
+
+echo "docker-entrypoint.sh here ... done"
```

Comment:
I think this line will be printed when the container stops, I think. But maybe that's what we want.

Reply:
Just some debug printing, I'll probably remove it at some point.